internal/cmd/operator-sdk/cli/cli.go

```diff
@@ -47,7 +40,6 @@ import (
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/scorecard"
 	"github.com/operator-framework/operator-sdk/internal/flags"
 	"github.com/operator-framework/operator-sdk/internal/plugins"
-	envtestv1 "github.com/operator-framework/operator-sdk/internal/plugins/envtest/v1"
```

Comment:
This is only used in the `v2` plugin, and that is no longer in `kubebuilder`.

Reply:
Clarity, as I'm understanding what I'm seeing, the envtest plugin here was basically an extra feature extending to go/v2 plugin bundle to make it more like the go/v3 plugin (at the time). With the jump to go/v4, the equivalent should be coming from the kubebuilder go/v4 scaffolding.

@acornett21 should envtest/v1 be pulled from this repo? Left in place, but marked as deprecated in some way?

Reply:
Alternatively, I see the "deprecated" notices attached to gov[23]Bundle. Should the strategy instead be "leave the deprecated bundles in place, but mark them deprecated" instead of "remove these"? I would take deprecation markings as an explicit vote to "keep these in place, so long as it's a low overhead maneuver to do so".

Reply:
Good question, I didn't notice there were no other references to this, since it's a plugin methods always show as used. Since the `v2` plugin is removed, I feel this should can also be removed.